src/types.ts

```diff
@@ -18,6 +18,7 @@ export type HookOptions = {
   onPickStart?: () => void
   onPickEnd?: () => void
   onPickCancel?: () => void
+  onChange?: (changes: OnChangeEyedrop) => void
```

Comment:
Hmm... I'm not sure about this.
Right now, the colors are returned from the hook itself on line 96.

Can't see any direct drawbacks with either way, but let's decide on one and get rid of the other. 🙌

Reply:
I think both are acceptable.  I see the colors more as a returned value to listen on (in the dev code), via a useEffect.  Whereas onChange is hit once.  It also gives more options.

Reply:
Alright then. Let's keep both! 🙌 🥳